plugins/base/src/main/kotlin/signatures/KotlinSignatureProvider.kt

```diff
@@ -113,6 +116,26 @@ class KotlinSignatureProvider(ctcc: CommentsToContentConverter, logger: DokkaLog
 
         is PrimitiveJavaType -> signatureForProjection(p.translateToKotlin())
     }
+
+    fun PageContentBuilder.DocumentableContentBuilder.funType(type: TypeConstructor) {
```

Comment:
We have a better DSL for that. Instead of extending `DocumentableContentBuilder` you should create an ordinary method and use it with unary plus: `+method()`

Reply:
Changed.